test/e2e/storage/drivers/csi-test/driver/driver.go

```diff
@@ -157,7 +157,7 @@ func goServe(server *grpc.Server, wg *sync.WaitGroup, listener net.Listener, sta
 		started <- true
 		err := server.Serve(listener)
 		if err != nil {
-			panic(err.Error())
```

Comment:
it seems that this should be using the `defer ginkgo.GinkgoRecover()`

Reply:
That would have made the panic show up as a Ginkgo failure instead of just being recorded in the log output. Panics shouldn't happen in this goroutine, therefore I am not sure whether it's worth it.